Rakefile

```diff
@@ -1,35 +1,16 @@
-require 'gem_publisher'
-
-desc 'Publish gem to RubyGems.org'
-task :publish_gem do |t|
-  gem = GemPublisher.publish_if_updated('logstash-output-kafka.gemspec', :rubygems)
-  puts "Published #{gem}" if gem
-end
+require "logstash/devutils/rake"
 
 task :default do
   system('rake -T')
 end
 
-desc "Get jars"
-task :vendor do
-  jar_target = "vendor/jar-dependencies/runtime-jars"
-  kafka_version = "0.9.0.1"
-  kafka_url = "http://central.maven.org/maven2/org/apache/kafka/kafka-clients/#{kafka_version}/kafka-clients-#{kafka_version}.jar"
-  slf4j_version = "1.7.13"
-  slf4j_url = "http://central.maven.org/maven2/org/slf4j/slf4j-api/#{slf4j_version}/slf4j-api-#{slf4j_version}.jar"
-  slf4j_nop_url = "http://central.maven.org/maven2/org/slf4j/slf4j-nop/#{slf4j_version}/slf4j-nop-#{slf4j_version}.jar"
-
-  puts "Will get jars for version #{kafka_version}"
-  puts "Removing current jars"
-  `rm -rf #{jar_target}`
-  `mkdir -p #{jar_target}`
-  Dir.chdir jar_target
-  puts "Will download #{kafka_url}"
-  `curl #{kafka_url} -o kafka-clients-#{kafka_version}.jar`
-  puts "Will download #{slf4j_url}"
-  `curl #{slf4j_url} -o slf4j-api-#{slf4j_version}.jar`
-  puts "Will download #{slf4j_nop_url}"
-  `curl #{slf4j_nop_url} -o slf4j-noop-#{slf4j_version}.jar`
+require 'jars/installer'
+task :install_jars do
+  # If we don't have these env variables set, jar-dependencies will
+  # download the jars and place it in $PWD/lib/. We actually want them in
+  # $PWD/vendor
+  ENV['JARS_HOME'] = Dir.pwd + "/vendor/jar-dependencies/runtime-jars"
+  ENV['JARS_VENDOR'] = "false"
```

Comment:
this needs to be true, but we also need to change "something" else. I will test it out real quick and get back to you on what "else" we need to change. the `JARS_HOME` environment variable needs to be set during runtime as well I think.  Otherwise `require_jar` command is lost and only checks the `.m2` cache for its dependencies